### PR TITLE
Player lib update

### PR DESCRIPTION
**Pull request by eamurao**

### CODE REVIEW

File: static/captivate/js/main.js [hunk 1]
```diff
@@ -40,10 +40,9 @@
     var customEllipsis = function () {
         var els = '.latest-video-title';
         var maxHeight = parseInt($(els).css('max-height')) + 1;
-        var lineHeight = isLoadMore ? 0 : parseInt($(els).css('line-height'));
```
Comment (rfornes) [inline static/captivate/js/main.js]: Is this change relevant for the player upgrade?

File: static/solo/js/libs/player.js [hunk 2]
```diff
@@ -183,12 +183,12 @@
         } else {
 
           //We create insntances on the tvpage player.
-          new TVPage.player({
+          that.player = new TVPage.player({
```
Comment (rfornes) [inline static/solo/js/libs/player.js]: Is this change relevant for the player upgrade?

File: static/solo-cta/js/libs/player.js [hunk 3]
```diff
@@ -216,12 +216,12 @@
                 } else {
 
                     //We create insntances on the tvpage player.
-                    new TVPage.player({
+                    that.player = new TVPage.player({
```
Comment (rfornes) [inline static/solo-cta/js/libs/player.js]: Is this change relevant for the player upgrade?

File: static/captivate/js/main.js (hunk 1 above)
Comment (eamurao) [inline static/captivate/js/main.js]: It's not relevant but this was raised by scrutinizer check and decided to include the fix here. Should I revert this fix?

Comment (rfornes) [inline static/captivate/js/main.js]: Ah ok, Is this like used variable thing? if so, is cool

File: static/solo/js/libs/player.js (hunk 2 above)
Comment (rfornes) [inline static/solo/js/libs/player.js]: same thing here?

Comment (eamurao) [inline static/solo/js/libs/player.js]: yes, this is also raised by scrutinizer check

File: static/solo-cta/js/libs/player.js (hunk 3 above)
Comment (eamurao) [inline static/solo-cta/js/libs/player.js]: and this one too...raised by scrutinizer check

File: static/captivate/js/main.js [hunk 4]
```diff
@@ -1471,7 +1470,7 @@
     if (TVSite.isPlayerPage) {
         window.TVPlayer = new TVPage.player({
             divId: 'TVPagePlayer',
-            swf: '//appcdn.tvpage.com//player/assets/tvp/tvp-1.5.2-flash.swf',
+            swf: '//cdnjs.tvpage.com/tvplayer/tvp-1.8.6.swf',
```
Comment (liorkuyer) [inline static/captivate/js/main.js]: Looks hard-coded

File: layouts/captivate/channelVideo.html [hunk 5]
```diff
@@ -14,6 +14,6 @@
   TVSite.channelVideosData.video = "undefined" !== typeof TVSite.channelVideosData.video || {};
   TVSite.channelVideosData.video = {{ $video | jsonify | safeJS }};
 </script>
-<script type="text/javascript" src="https://appcdn.tvpage.com/player/assets/tvp/tvp-1.8.5-min.js"></script>  
+<script type="text/javascript" src="https://cdnjs.tvpage.com/tvplayer/tvp-1.8.6.min.js"></script>
```
Comment (liorkuyer) [inline layouts/captivate/channelVideo.html]: Looks hard-coded

**Pull request closed without merging**